stream/stream_bluray.c

```diff
@@ -465,13 +478,16 @@ static int bluray_stream_open(stream_t *s)
         b->cfg_title = BLURAY_DEFAULT_TITLE;
     } else if (bstr_equals0(title, "menu")) {
         b->cfg_title = BLURAY_MENU_TITLE;
+    } else if (bstr_equals0(title, "mpls")) {
+        bstr_split_tok(bdevice, "/", &title, &bdevice);
+        b->cfg_playlist = bstrtoll(title, &rest, 10);
```

Comment:
Yes, you are right, I added a range check to the title index and playlist. Previously by specifying an title index of -3 an access to uninitialized memory could be caused.